usagov/vote-gov
# Vote-721: Update to NY Deadline

Ticket: [Vote-721](https://cm-jira.usa.gov/browse/VOTE-721)
Purpose:  This PR will updated the deadline for NY from 25 to 10 
Testing:

1. check file changes to send all files are updated as expected 
2. visit [preview link](https://cg-9e8debaf-b030-4825-a43c-cb2bc850c96c.sites.pages.cloud.gov/preview/usagov/vote-gov/feature/vote-721-ny-date-update/register/ny/) and check that there is no visual regression in update
